### PR TITLE
Block share

### PR DESCRIPTION
**Pull request by dzucconi**
Closes https://github.com/aredotna/ervell/issues/1026

### CODE REVIEW

File: apps/block/components/share/index.coffee [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+module.exports = ({ block }) ->
+  $el = $('.js-block-share')
+
+  view = new BlockShareView
+    text: generateText(block.toJSON())
+
+  $el.html view.render().$el
+
+  view
```
Comment (dzucconi) [inline apps/block/components/share/index.coffee]: Woops lemme make sure this gets cleaned up in block views.

**Comment by dzucconi** [comment]
Good lord this is ugly.

File: apps/block/components/share/index.coffee (hunk 1 above)
Comment (broskoski) [reply apps/block/components/share/index.coffee]: Cleaned up as in torn down when BlockView gets removed?

Comment (dzucconi) [reply apps/block/components/share/index.coffee]: Yes. which now it does.

**Pull request merged**